<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow2</class>
 <widget class="QMainWindow" name="MainWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controller</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/images/docs/emblem-web.png</normaloff>:/images/docs/emblem-web.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabListeVisite">
       <attribute name="title">
        <string>All visits</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;All visits :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidgetVisites">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <column>
             <property name="text">
              <string>predicted date</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>producer</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>administrator</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>type visit</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>condition</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ground refusal</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Final choice</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGestionVisite">
       <attribute name="title">
        <string>Visit management</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Visit management :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Producer :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelProducteurAtt">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Administrator :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelGestionnaireAtt">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>predicted date :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDatePrevueAtt">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Type visit :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelTypeVisiteAtt">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Condition :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBoxEtatVisite">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string>ATT</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ACC</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>REF</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_12">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Reason for refusal sent to manager :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTextEdit" name="textEditMotifRefus">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonValiderAtt">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="ressources.qrc">
                <normaloff>:/images/docs/add.png</normaloff>:/images/docs/add.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGestionVisiteAccepter">
       <attribute name="title">
        <string>Management visits accept</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Management visit :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QLabel" name="label_16">
                    <property name="text">
                     <string>Predicted date :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelDatePrevueAcc">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QLabel" name="label_15">
                    <property name="text">
                     <string>Type visit :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelTypeVisiteAcc">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_14">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Producer :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelProducteurAcc">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_15">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Administrator :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelGestionnaireAcc">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="line_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_13">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Final choice :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTextEdit" name="textEditChoixFinal">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLabel" name="label_14">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Condition of producer :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBoxEtatProducteur">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <item>
                     <property name="text">
                      <string>ACC</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>REF</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonValiderAcc">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="ressources.qrc">
                <normaloff>:/images/docs/add.png</normaloff>:/images/docs/add.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControleur">
    <property name="title">
     <string>Controller</string>
    </property>
   </widget>
   <addaction name="menuControleur"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
